packages/agents/src/index.ts

```diff
@@ -759,6 +780,94 @@ export class Agent<Env, State = unknown> extends Server<Env> {
     // biome-ignore lint/complexity/noBannedTypes: <explanation>
     return callableMetadata.has(this[method as keyof this] as Function);
   }
+
```

Comment:
I'm hesitant to add this to root Agent class, tho I might be able to be convinced. Ping me and let's chat about it?